pkg/sources/openshift.com/mirror/mirror_test.go

```
@@ -0,0 +1,33 @@
+package mirror
+
+import (
+	"net/url"
+	"testing"
+)
+
+// The test is to verify that NewSource correctly intializes the mirror source.
+func TestNewSource(t *testing.T) {
```

golangci-lint (2.12.2)

[error] Function TestNewSource missing the call to method parallel


(paralleltest)